jsk_fetch_robot/jsk_fetch_startup/scripts/odometry_transformer.py

```diff
@@ -67,7 +68,9 @@ def cb_odometry(self, msg):
            math.isnan( msg.twist.twist.angular.x ) or \
            math.isnan( msg.twist.twist.angular.y ) or \
            math.isnan( msg.twist.twist.angular.z ):
-            rospy.logwarn('Recieved an odom message with nan values')
+            if (rospy.Time.now() - self._last_warning).secs > 10:
+                rospy.logwarn('Received an odom message with nan values')
+                self._last_warning = rospy.Time.now()
```

Comment:
you can use `rospy.logwarn_throttle`
http://wiki.ros.org/rospy/Overview/Logging#Logging_Periodically